python-runnables/resize-node-pool/runnable.py

```diff
@@ -47,25 +47,14 @@ def run(self, progress_callback):
         if autoscaling_enabled:
             min_nodes = self.config['minNumNodes']
             max_nodes = self.config['maxNumNodes']
-            initial_node_count = self.config['numNodes']
-            if min_nodes is None:
-                raise Exception("Cannot make autoscalable cluster with no minimum number of nodes.")
-            elif max_nodes is None:
-                raise Exception("Cannot make autoscalable cluster with no maximum number of nodes.")
-            elif initial_node_count is None or initial_node_count < min_nodes or initial_node_count > max_nodes:
-                raise Exception("The initial number of nodes must be set and between the min and max number of nodes bounds ([%s %s])."
-                                % (min_nodes, max_nodes))
-            elif min_nodes > max_nodes:
-                raise Exception("Cannot make autoscalable cluster with a max number of nodes (%s) less than its min number of nodes (%s)."
+            if min_nodes > max_nodes:
+                raise Exception("Cannot resize autoscalable cluster with a max number of nodes (%s) less than its min number of nodes (%s)."
                                 % (max_nodes, min_nodes))
             else:
-                logging.info("Resizing cluster to autoscale with %s min nodes and %s max nodes and an initial node count of %s."
-                             % (min_nodes, max_nodes, initial_node_count))
+                logging.info("Resizing cluster to autoscale with %s min nodes and %s max nodes."
+                             % (min_nodes, max_nodes))
                 node_pool.min_count = min_nodes
-                node_pool.count = initial_node_count
```

Comment:
With python 2.7 do we also need here to set node_pool.count = None?
I wonder if it would be possible to have a failure if the old initial_node_count is outside of the bound of nim/max?
I'm will provision an Azure machine an double check that

Reply:
I wasn't able to find any issue here sso we are good to go.